src/Chainweb/Pact/Types.hs

```diff
@@ -798,9 +798,7 @@ data LocalPreflightSimulation
     deriving stock (Eq, Show, Generic)
 
 -- | The type of local results (used in /local endpoint)
--- Internally contains a JsonText instead of a CommandResult for compatibility across Pact versions,
--- but the constructors are hidden.
--- This can be undone in the 2.28 release.
```

Comment:
I suggest leaving the note about changing this back in the 2.28 release; there is no need for compat with Pact 4 after it becomes impossible to use Pact 4 in /local.